storage/storage.rs

```diff
@@ -206,18 +215,50 @@ impl<D> MVCCStorage<D> {
         D: DurabilityService,
     {
         use StorageRecoverError::DurabilityServiceRead;
-        let records = self
-            .durability_service
-            .iter_type_from_start::<CommitRecord>()
+        let records =
+            IsolationManager::iterate_commit_status_from_disk(&self.durability_service, SequenceNumber::MIN)
             .map_err(|error| DurabilityServiceRead { source: error })?;
         for record in records {
             match record {
-                Ok((commit_sequence_number, commit_record)) => {
-                    self.try_write_commit_record(commit_sequence_number, commit_record).unwrap();
-                }
+                Ok(commit_status) => {
+                    match commit_status {
+                        // TODO: Relative indexes once sequence numbers become discontinuous
+                        CommitStatus::Applied(commit_sequence_number, commit_record) => {
+                            self.isolation_manager.load_applied(
+                                Self::tmp_relative_index_from_sequence_number(commit_sequence_number),
+                                commit_sequence_number,
+                                commit_record.into_owned(),
+                            );
+                        }
+                        CommitStatus::Closed(commit_sequence_number) => {
+                            self.isolation_manager.load_aborted(
```

Comment:
If you're going to switch to the 'aborted' terminology, you should do it consistently - `CommitStatus::Aborted`